src/_guides/language/sound-dart.md

```diff
@@ -736,15 +737,16 @@ strong mode:
   to set up and customize the analyzer and linter using an analysis
   options file.
 
-The next few documents are part of the Dart Dev Compiler (DDC) documentation,
+The next few documents are part of the original
+[dartdevc]({{site.webdev}}/tools/dartdevc) documentation,
 but most of the information applies to anyone using strong mode Dart:
 
 * [Strong Mode](https://github.com/dart-lang/dev_compiler/blob/master/STRONG_MODE.md) -
   Motivation for strong mode Dart.
 * [Strong Mode Static Checking](https://github.com/dart-lang/dev_compiler/blob/master/doc/STATIC_SAFETY.md) -
   Type inference in strong mode Dart.
 * [Strong Mode in the Dart Dev Compiler](https://chromium.googlesource.com/external/github.com/dart-lang/dev_compiler/+/refs/heads/master/doc/RUNTIME_SAFETY.md) -
-  Runtime checks in DDC.
+  Runtime checks in dartdevc.
 * [Prototype Syntax for Generic Methods](https://github.com/dart-lang/dev_compiler/blob/master/doc/GENERIC_METHODS.md) -
```

Comment:
OK, I'll rename that link. I should probably link (earlier) to https://www.dartlang.org/guides/language/language-tour#using-generic-methods